### PR TITLE
feat(patreon): add message for patreon users who do not support the project

### PR DESCRIPTION
**Pull request by evakill**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description
it seems like we should have a message for people who a) connect with patreon but b) are not an active supporter

added some thorough tests for this component

removed beta wrapper - ready to release! 

## Git Issues

Closes #

## Screenshots/Videos
@davehakkens let me know if you like the looks of this?
Screenshot 2024-01-08 at 7 03 43 PM

_If useful, provide screenshot or capture to highlight main changes_

---

## What happens next?

Thanks for the contribution! We try to make sure all PRs are reviewed ahead of our monthly maintainers call (first Monday of the month)

If the PR is working as intended it'll be merged and included in the next platform release, if not changes will be requested and re-reviewed once updated.

If you need more immediate feedback you can try reaching out on Discord in the [Community Platform `development` channel](https://discord.com/channels/586676777334865928/938781727017558018).

### CODE REVIEW

**Comment by davehakkens** [comment]
Nice one. Sounds good to me. Lets roll it out and see if it does the job! :)

File: src/pages/UserSettings/content/formSections/PatreonIntegration.tsx [hunk 1]
```diff
@@ -8,24 +8,26 @@ import { FlexSectionContainer } from './elements'
...
   'Support us on Patreon to get a badge here on the platform and special insights and voting rights on decisions.'
 const BETA_DISCLAIMER =
   'This feature is still in beta and we will continue to roll out more features for supporters.'
-const SUCCESS_MESSAGE = 'Successfully linked Patreon account!'
-const SUPPORTER_MESSAGE =
+export const SUCCESS_MESSAGE = 'Successfully linked Patreon account!'
+export const SUPPORTER_MESSAGE =
   'Thanks for supporting us! :) Update your data if you changed your Patreon tiers or remove the connection below.'
+export const BECOME_A_SUPPORTER_MESSAGE =
+  'Thanks for connecting your account! It looks like you are not an active supporter of this project. If you become a supporter in the future, click the button below to update your data.'
```
Comment (iSCJT) [inline src/pages/UserSettings/content/formSections/PatreonIntegration.tsx:20]: (question): do we want to give them more of a call to action with a link through to the page to become a supporter?

**Review by iSCJT — COMMENTED** [review]
Looks good to me 👍  Only question I have is whether we update the text further to try to encourage people to become a supporter?

File: src/pages/UserSettings/content/formSections/PatreonIntegration.tsx (hunk 1 above)
Comment (evakill) [reply src/pages/UserSettings/content/formSections/PatreonIntegration.tsx:20]: good idea! will add

**Pull request merged**